hibernate6/src/main/java/com/fasterxml/jackson/datatype/hibernate6/Hibernate6ProxySerializer.java

```diff
@@ -249,6 +249,7 @@ private String getIdentifierPropertyName(final LazyInitializer init) {
         if (_mapping != null) {
             idName = _mapping.getIdentifierPropertyName(init.getEntityName());
         } else {
+            // no unit tests rely on this next call and Hibernate 7 does not support it
```

Comment:
Is this change related to Hibernate 7 addition? If not (and I suspect it isn't), could we have separate PR to merge this change in first?

Reply:
Merged, updated this PR.